lib/types/operation.js

```diff
@@ -90,7 +90,7 @@ var Operation = module.exports = function (parent, scheme, operationId, httpMeth
     if (innerType && innerType.toString().toLowerCase() === 'boolean') {
       param.allowableValues = {};
       param.isList = true;
-      param['enum'] = ['true', 'false'];
+      param['enum'] = [true, false]; // use actual primitives
```

Comment:
Maybe there is code elsewhere in the converter that already handles this.  If you're sure it's fine, I've no reason to push back.  :)